var/spack/repos/builtin/packages/mysql/package.py

```diff
@@ -105,8 +119,34 @@ def cmake_args(self):
             options.append('-DWITHOUT_SERVER:BOOL=ON')
         return options
 
+    def fix_dtrace_usr_bin_path(self, spack_env):
+        # dtrace may cause build to fail because it uses
+        # '/usr/bin/python' in the shebang. To work around that we copy
+        # the original script into a temporary folder, and change the
+        # shebang to '/usr/bin/env python'.
```

Comment:
Should this be `/usr/bin/env python` or `/bin/env python`?  (I don't know the answer, hoping someone will).

Reply:
There is no `/bin/env`, only `/usr/bin/env`. But it might be better to use Spack's Python explicitly in the shebang. Several other packages do this. @hartzell has done this on several Perl packages as well.

Reply:
@citibeth Sorry for letting this drop: deadlines and vacation intervened.  Per https://github.com/spack/spack/pull/11356/commits/6934fe11de84dcd8df3eeb1969cf12ce060dda40, I have added a comment specifying the origin of this code as being @alalazo's improvement of the `glib` recipe.

@adamjstewart I can do this here if you desire, but given that this change should probably be made to `glib` also would this be better as the subject of a distinct PR to follow?

Reply:
@citibeth, @adamjstewart please let me know whether your concerns have been addressed.